src/Native/Common/pal_config.h.in

```diff
@@ -21,9 +21,12 @@
 #cmakedefine01 HAVE_FDS_BITS
 #cmakedefine01 HAVE_PRIVATE_FDS_BITS
 #cmakedefine01 HAVE_ALLOCA_H
+#cmakedefine01 HAVE_PTHREAD_H
```

Comment:
We assume pthreads is available in other places in the system.  On what platforms we're currently thinking about is it not available?  If this is being done for future-proofing rather than because it's needed today, I'd much rather wait to add such feature detection until it's deemed actually necessary.